ext/node/ops/require.rs

```diff
@@ -575,32 +568,35 @@ where
 #[string]
 pub fn op_require_package_imports_resolve<P>(
```

Comment:
Needed to resolve the closest package.json (instead of the just one in the parent folder) and convert the URL to a path before returning.